src/protocols/kubernetes/argv.h

```diff
@@ -37,5 +37,26 @@
  */
 guac_user_argv_handler guac_kubernetes_argv_handler;
 
+/**
+ * Sends the current values of all non-sensitive parameters which may be set
+ * while the connection is running to the given user. Note that the user
+ * receiving these values will not necessarily be able to set new values
+ * themselves if their connection is read-only. This function is provided for
+ * convenience, as it is can be used as the callback for
```

Comment:
This sentence seems a little odd to me - "and can be used" or "it can be used" or "as it is it can be used"?

Reply:
I'll switch things from "This function is provided for convenience, as it is can be used as the callback for ..." to "This function can be used as the callback for ...".

The caller can decide for themselves whether it's convenient. ;)